src/settings.ts

```diff
@@ -2,12 +2,21 @@ export enum METRIC_NAMES {
   ANCHOR_SUCCESS = 'anchor_success',
   ANCHOR_TIMEOUT = 'anchor_timeout',
```

Comment:
It seems, the file is only metrics related. Maybe reflect that in name, rather than use generic `settings`?

Reply:
IDK, I see no other settings for now, so your proposal for renaming makes all sense to me. IMO, it makes sense to use dash instead of underscore also, to be uniform with other places in code.